### DISCUSSION TITLE
Identifier of lustre grammar in KIND2

### DISCUSSION THREAD

**Discussion by kumori123** [kind2-mc/kind2#925] [Q&A] [answered]
Hi, I'm now trying to deal with converting lustre v6 to lustre which could be recognized by KIND2. Due to the recursive structure of lustre v6, like 
```
package1
        Operator1
        Package2
                 Operator1
```
I intended to rewrite the Operator name like "Operator1" -> "package1_Operator1" and so on, but I realised that it may cause disambiguation with variables with underline. 
Do you have any suggestions?
Thank you!

**Comment by daniel-larraz** [comment, marked as answer]
Hi, an easy solution to make each identifier unique is to use a counter. Just prepend a prefix to the new identifier with the current value of the counter.

Best,
Daniel

**Reply by kumori123** [reply]
Thank you！
